### PR TITLE
Add API to get module event 

### PR DESCRIPTION
**Pull request by 0xaptosj**
### Description

Add new event API to get module event from indexer.

### Test Plan

Uint test.

### Related Links

https://github.com/aptos-labs/aptos-developers/discussions/6

### CODE REVIEW

**Review by alinush — APPROVED** [review]
Blind stamp

File: src/api/event.ts [hunk 1]
```diff
@@ -15,6 +20,27 @@ import { waitForIndexerOnVersion } from "./utils";
...
+  /**
+   * Get module events by event type
+   *
+   * @param args.eventType - The event type
+   * @param args.minimumLedgerVersion Optional ledger version to sync up to, before querying
+   *
+   * @returns Promise<GetEventsResponse>
+   */
+  async getModuleEventsByEventType(args: {
+    eventType: MoveStructId;
```
Comment (bowenyang007) [inline src/api/event.ts:32]: Can you add a check here that if the event type is too long (wrapped for example) we should send a warning and possibly error. We cap indexed_event at length 300.

**Review by 0xmaayan — APPROVED** [review]
LGTM. Please update changelog

File: src/api/event.ts (hunk 1 above)
Comment (0xaptosj) [reply src/api/event.ts:32]: Added to [internal function](https://github.com/aptos-labs/aptos-ts-sdk/pull/304/files#diff-e2ce900e3281d047225ea1fd842abcb5fa55614f33438d27e1b1b6e342127ccbR101)

**Pull request merged**